docs/learn-phragmen.md

```diff
@@ -26,27 +26,382 @@ which are elected.
+3. Now we start electing candidates.  For the list of all candidates who have not been elected, get their score, which is equal to `1 / approval_stake`.
+4. For each voter, update the score of each candidate they support by adding their total budget (stake) multiplied by the load of the candidate and then dividing by that candidate's approval stake (`voter_budget * voter_load / candidate_approval_stake`.
+5. Determine the candidate with the lowest score and elect that candidate.  Remove the elected candidate from the pool of potential candidates.
+6. The load for each edge connecting to the winning candidate is updated, with the edge load set to the score of the candidate minus the voter's load, and the voter's load then set to the candidate's score.
+7. If there are more candidates to elect, go to Step 3.  Otherwise, continue to step 8.
+8. Now the stake is distributed amongst each nominator who backed at least one elected candidate.  The backing stake for each candidate is calculated by taking the budget of the voter and multiplying by the edge load then dividing by the candidate load (`voter_budget * edge_load / candidate_load`).
+
+### Example
+
+_Note: floating-point numbers in this example are rounded off to three decimal places._
+
+In the following example, there are five voters and five candidates vying for three potential seats.  Each voter `V1 - V5` has an amount of stake equal to their number (e.g., `V1` has stake of 1, `V2` has stake of 2, etc.).  Every voter is also going to have a _load_ which initially starts at `0`.
+
+```
+Filled seats: 0
+Open Seats: 3
+
+Candidates:    A B C D E  L0
+----------------------------
+Voter V1 (1):  X X        0
+Voter V2 (2):  X X        0
+Voter V3 (3):  X          0
+Voter V4 (4):    X X X    0
+Voter V5 (5):  X     X    0
+```
+
+Let us now calculate the approval stake of each of the candidates.  Recall that this is merely the amount of all support for that candidate by all voters.
+
+```
+Candidate A: 1 + 2 + 3 + 5 = 11
+Candidate B: 1 + 2 + 4 = 7
+Candidate C: 4 = 4
+Candidate D: 4 + 5 = 9
+Candidate E: 0
+```
+
+The first step is easy - candidate `E` has 0 approval stake and can be ignored from here on out.  They will never be elected.
+
+We can now calculate the initial scores of the candidates, which is `1 / approval_stake`:
+
+```
+Candidate A: 1 / 11 = 0.091
+Candidate B: 1 / 7 = 0.143
+Candidate C: 1 / 4 = 0.25
+Candidate D: 1 / 9 = 0.111
+Candidate E: N/A
+```
+
+For every edge, we are going to calculate the score, which is current score plus the total budget * the load o the voter divided by the approval stake of the candidate.  However, since the load of every voter starts at 0, and anything multiplied by 0 is 0, any addition will be `0 / x`, or 0.  This means that this step can be safely ignored for the initial round.
```

Comment:
```suggestion
For every edge, we are going to calculate the score, which is current score plus the total budget * the load of the voter divided by the approval stake of the candidate.  However, since the load of every voter starts at 0, and anything multiplied by 0 is 0, any addition will be `0 / x`, or 0.  This means that this step can be safely ignored for the initial round.
```